file-events/src/test/groovy/net/rubygrapefruit/platform/file/BasicFileEventFunctionsTest.groovy

```diff
@@ -417,15 +415,18 @@ class BasicFileEventFunctionsTest extends AbstractFileEventFunctionsTest {
     def "fails when watching non-existent directory"() {
         given:
         def missingDirectory = new File(rootDir, "missing")
+        def message = Platform.current().linux
+            ? "Couldn't stat path"
+            : "Couldn't add watch"
```

Comment:
I wonder if we should standardize the messages themselves instead.

Reply:
Yes, I think it would be better to standardize here. The message should probably be `Couldn't add watch`, since that what we tried to do. The user of the API doesn't know why we needed to stat the thing we wanted to watch.

Reply:
Agreed, though it would make sense to somehow also capture that the error message belongs to `stat()` (as opposed to `inotify_add_watch()`) for disambiguation later. So something like:

- `Couldn't add watch (stat() failed with errno = 2)`
- `Couldn't add watch (inotify_add_watch() failed with errno = 3)`

WDYT?

Reply:
Sounds good. I'll make the change.

Reply:
Done.